scripts/collect-info.yaml

```diff
@@ -7,11 +7,9 @@
     debug: false
     commands:
       - name: git_info
-        cmd: cat {kargo,.}/.git/logs/HEAD
+        cmd: find . -type d -name .git -execdir sh -c 'gen-gitinfos.sh global|head -12' \;
```

Comment:
This will match all git dirs on a given host. For now, it's okay, but maybe we want it to match regex /kargo$/